components/admin/billing/AdminBillingPage.tsx

```
@@ -0,0 +1,828 @@
+
+    try {
+      switch (action) {
+        case 'send':
+          newStatus = InvoiceStatus.SENT;
+          await apiUpdateInvoice(invoiceId, { status: newStatus });
+          successMessage = 'Invoice marked as Sent.';
+          break;
+        case 'mark_paid':
+          newStatus = InvoiceStatus.PAID;
+          const invoiceToPay = invoices.find(inv => inv.id === invoiceId);
+          if (invoiceToPay) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)